app/controllers/users_controller.rb

```diff
@@ -207,9 +207,7 @@ def set_all_configurations
                                    .includes(:motif_category)
                                    .where(current_organisation_filter)
                                    .uniq(&:motif_category_id)
-
-    @all_configurations =
-      department_level? ? @all_configurations.sort_by(&:department_position) : @all_configurations.sort_by(&:position)
+    @all_configurations.sort_by { |c| department_level? ? c.department_position : c.position }
```

Comment:
Je crois qu'on doit faire ça du coup non ?
```suggestion
    @all_configurations.sort_by! { |c| department_level? ? c.department_position : c.position }
```

Reply:
ah yes bien vu, my bad